src/Bundle/CoreBundle/Resources/config/routing.subdomain.yml

```diff
@@ -0,0 +1,149 @@
+unitecms_core_index:
+    path:     /
+    methods: ['GET']
+    host:     'profile.{hostname}'
+    requirements:
+        hostname: '%allowed_hostname%'
+    defaults:
+        _controller: UniteCMSCoreBundle:Index:index
+        hostname: '%default_hostname%'
+
+unitecms_core_authentication:
+    resource: "@UniteCMSCoreBundle/Controller/AuthenticationController.php"
+    prefix:   /
+    type:     annotation
+    host:     'profile.{hostname}'
+    requirements:
+        hostname: '%allowed_hostname%'
+    defaults:
+        hostname: '%default_hostname%'
+
+unitecms_core_logout:
+    path: /logout
+    host:     'profile.{hostname}'
+    requirements:
+        hostname: '%allowed_hostname%'
+    defaults:
+        hostname: '%default_hostname%'
+
+unitecms_core_profile:
+    resource: "@UniteCMSCoreBundle/Controller/ProfileController.php"
+    prefix:   /
+    type:     annotation
+    host:     'profile.{hostname}'
+    requirements:
+        hostname: '%allowed_hostname%'
+    defaults:
+        hostname: '%default_hostname%'
+
+
+unitecms_core_organizations:
+    resource: "@UniteCMSCoreBundle/Controller/OrganizationController.php"
+    type:     annotation
+    prefix:   /organizations
+    host:     'profile.{hostname}'
+    requirements:
+        hostname: '%allowed_hostname%'
+    defaults:
+        hostname: '%default_hostname%'
+
+unitecms_core_organization_user:
+    resource: "@UniteCMSCoreBundle/Controller/OrganizationUserController.php"
+    type:     annotation
+    prefix:   /user
+    host:     '{organization}.{hostname}'
+    requirements:
+        organization: '[a-z0-9_]+'
+        hostname: '%allowed_hostname%'
```

Comment:
wouldn't this be a cleaner approach? instead of having this regex in here multiple times
https://symfony.com/doc/current/routing/service_container_parameters.html

Reply:
do you mean to the regex for the organization parameter?

We could also put this regex in a const (e.g. Organizaton::IDENTIFIER_PATTERN and use it for the config as well as for the entity validation regex. What do you say?

Reply:
yep, the organization parameter
a const would do the job as well